dipy/segment/mrf.pyx

```diff
@@ -217,48 +239,7 @@ class ConstantObservationModel(object):
         return mu_upd, var_upd
 
 
-    def update_param_new(self, image, P_L_Y, mu, nclasses):
```

Comment:
It seems this was not used anywhere @grlee77 ?

Reply:
Yes, it seems to be some leftover dead code. I did a quick grep of the code base just to be sure and don't see any other files that refer to it.